content/en/network_monitoring/devices/network_topology_map.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Network Topology Map
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/network_monitoring/devices/data"
+  tag: "Documentation"
+  text: "Data Collected with Network Device Monitoring"
+- link: "https://www.datadoghq.com/blog/monitor-snmp-with-datadog/"
+  tag: "Blog"
+  text: "Monitor SNMP with Datadog"
+---
+
+<div class="alert alert-warning">The Network Topology Map for Datadog Network Device Monitoring is in public beta.</div>
+
+## Overview
+
+The Network Topology map provides an overview of your network's physical connections, so you can more easily identify issues in your devices and understand their upstream and downstream impacts. 
```

Comment:
Could add a link to the UI or navigations steps, so users know how to find this feature.